### DISCUSSION TITLE
Conan

### DISCUSSION THREAD

**Discussion by Etienne-P54** [uNetworking/uWebSockets#1172] [General]
Hello,

I'm currently developping a large C++ project and I would like to use uWebSockets to expose a REST API.
I'm using Conan to handle all the dependencies. Unfortunatly, the current version of uWebSockets on Conan is 18.3.0 which is fairly old.
Do you have any plan to update the Conan package on a regular basis ?
Thanx,

Etienne

**Comment by ghost** [comment]
Hi,

I don't publish anything to Conan, so you have to contact whomever did.

**Reply by Etienne-P54** [reply]
ok thanx for the quick reply
